openedx/core/djangoapps/schedules/management/commands/tests/test_send_email_base_command.py

```diff
@@ -36,10 +36,10 @@ def test_handle(self):
                 None
             )
 
-    def test_duration_option(self):
+    def test_weeks_option(self):
         with patch.object(self.command, 'enqueue') as enqueue:
-            self.command.handle(site_domain_name=self.site.domain, date='2017-09-29', duration=63)
-            self.assertEquals(enqueue.call_count, 8)
+            self.command.handle(site_domain_name=self.site.domain, date='2017-09-29', weeks=3)
+            self.assertEquals(enqueue.call_count, 3)
```

Comment:
nit: Since the client's reason for wanting this change is to allow there to be >10 weeks of emails sent (which is the default), could this test verify using `weeks=12`?

Reply:
Done